package.json

```diff
@@ -18,7 +18,8 @@
     }
   },
   "lint-staged": {
-    "*": "yarn format:nopath"
+    "src/**/*": "yarn lint --fix",
+    "plugins/*": "yarn lint --fix"
```

Comment:
wondering if it would make more sense to trigger on all .js files and use .eslintignore instead 🤔 thoughts?

also the `format:nopath` option was in there because lint-staged will automatically pass the path to the end of the argument, this makes it so that it will only run on the files that are changed / staged, not the entire codebase. i think we'd still want that feature

Reply:
also so with the way this is set up, this will use eslint for fixing, but it will just bring in prettier rules? is there any benefit for using eslint for the fixing vs using prettier for the fixing? mostly just asking out of curiosity, ive done it the other way around before https://github.com/colbyfayock/gatsby-starter-leaflet/blob/master/package.json#L64